src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -0,0 +1,22 @@
+# DO NOT CHANGE THESE VALUES
+SHARED_METRIC_NAMES = {
+    "c:sessions/session@none": 1,
+    "s:sessions/error@none": 2,
+    "s:sessions/user@none": 3,
+    "d:sessions/duration@second": 4,
+}
```

Comment:
It might not matter technically, but should we leave some space between the last metric name (4) and the first tag (5), just in case we get more shared metric names in the future? Or would it be a better idea to reserve ranges per use case, e.g. `0..99` for sessions, `100..199` for transactions?

Reply:
yeah was thinking about this too, seems easy enough to split into ranges